common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4691,6 +4694,15 @@ object CelebornConf extends Logging {
       .bytesConf(ByteUnit.BYTE)
       .createWithDefaultString("64k")
 
+  val PARTITION_READER_CHECKPOINT_ENABLED: ConfigEntry[Boolean] =
+    buildConf("celeborn.partition.reader.checkpoint.enabled")
```

Comment:
Maybe rename the configuration to `celeborn.client.partition.reader.checkpoint.enabled`, since it is only for client use.

Reply:
Ack